### ISSUE TITLE
'lmfit.models' has no attribute 'DoniachModel'

### ISSUE THREAD

**Issue by maxdevblock** [lmfit/lmfit-py#639]
#### Description
Error defining DoniachModel

###### A Minimal, Complete, and Verifiable example
```python
import lmfit

model = lmfit.models.DoniachModel()
```

###### Error message:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-64-8090aa95b5f2> in <module>
----> 1 model = lmfit.models.DoniachModel()

AttributeError: module 'lmfit.models' has no attribute 'DoniachModel'
```

###### Version information
Conda environment on macOs Catalina.
```
Python: 3.7.7 (default, Mar 26 2020, 10:32:53) 
[Clang 4.0.1 (tags/RELEASE_401/final)]

lmfit: 1.0.0, scipy: 1.4.1, numpy: 1.18.1, asteval: 0.9.16, uncertainties: 3.1.2
```

**Comment by newville** [comment]
fixed with  f3af0d4c052ca8ae4a4927581ba485c659f18d3c !  It was spelled incorrectly as `DonaichModel()`

**Comment by maxdevblock** [comment]
> fixed with [f3af0d4](https://github.com/lmfit/lmfit-py/commit/f3af0d4c052ca8ae4a4927581ba485c659f18d3c) ! It was spelled incorrectly as `DonaichModel()`

Ah sorry! My fault.
Copy and paste from documentation
https://lmfit.github.io/lmfit-py/builtin_models.html#doniachmodel
You may want to check spell there :-)
